test/show.jl

```diff
@@ -1010,6 +1010,9 @@ test_mt(show_f5, "show_f5(A::AbstractArray{T, N}, indices::Vararg{$Int, N})")
 @test sprint(show, :(function f end)) == ":(function f end)"
 @test_repr "function g end"
 
+# Printing of :(function (x...) end)
+@test replstr(Meta.parse("function (x...) end")) == ":(function (x...,)\n      #= none:1 =#\n      #= none:1 =#\n  end)"
```

Comment:
This tests seems a bit brittle with all the line info stuff. Maybe using something like `startswith` would be good to isolate the actual function expression printing?

Reply:
I have just copied this from some other test in the same file, but I agree that using `startswith` definitely seems more robust.